test/abbreviation_bug_test.py

```diff
@@ -48,6 +48,8 @@ def test(host, pguser):
     assert( os.path.isfile(sqlite_test_filename) and "sqlite file must exist at this point" )
 
     scon = dbapi2.connect(sqlite_test_filename)
+    scon.enable_load_extension(True)
+    scon.execute("SELECT load_extension('mod_spatialite')")
```

Comment:
You should try to do manual tests of the plugin (not only this tests) on windows, I remember the problems in doing that, but maybe it's better now.

Reply:
I try on windows and it's work!